### PR TITLE
docs: ui.text

### LINKED ISSUE #835: docs: ui.text

### PR DESCRIPTION
**Pull request by AkshatJawne**
Closes #835

**Additional Info:**
- Link to markdown once implementation is finished

### CODE REVIEW

File: plugins/ui/docs/components/text.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+from deephaven import ui
+
+my_text_basic = ui.text("Paste")
+```
+
+## UI Recommendations
+
+If single-line input is required, consider using [`text_field`](./text_field.md). If multi-line input is required, use [`text_area`](./text_area.md).
+
+
```
Comment (dsmmcken) [inline plugins/ui/docs/components/text.md:17]: I think the only unusual prop is color (something we added not spectrum).

Should show it as an example.

```
faded_text = ui.text("Faded text", color="gray-500")
negative_text = ui.text("Faded text", color="negative")
postive_text = ui.text("Faded text", color="positive")
hex_text = ui.text("Hex color", color="#FA8072")
sentence_with_color = ui.text("Hello ", ui.text("world", color="accent"), "!")
```

File: plugins/ui/docs/components/text.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
+# Text
+
+Text represents text with no specific semantic meaning, serving as a basic element.
```
Comment (dsmmcken) [inline plugins/ui/docs/components/text.md:3]:
```suggestion
Text represents text with no specific semantic meaning, serving as a basic element. It is equivalent to an HTML span element.
```

**Review by dsmmcken — CHANGES_REQUESTED** [review]

File: plugins/ui/docs/components/text.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+```
+
+## UI Recommendations
+
+Consider using [`ui.heading`](./heading.md) if you want to create different types of headings.
+
+
+## Content
+
+The text component is sometimes used within a parent container, like [`button`](./button.md) or [`picker`](./picker.md) to define text content when multiple children are passed (such as icons and descriptions) . It inherits styling from its parent container.
```
Comment (dsmmcken) [inline plugins/ui/docs/components/text.md:20]:
```suggestion
The text component is sometimes used within a parent container, like [`button`](./button.md) or [`picker`](./picker.md) to define text content when multiple children are passed (such as icons and descriptions). It inherits styling from its parent container.
```

**Review by AkshatJawne — COMMENTED** [review]
Added to sidebar.json @dsmmcken , should be good for re-review

**Review by dsmmcken — APPROVED** [review]

**Pull request merged**